<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JFC调试器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>设备</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_srvPort">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>53</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>00000</string>
     </property>
     <property name="text">
      <string>9734</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_srvIp">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>23</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.10.139</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>23</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>设备IP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEdit_srvIp</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>52</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>设备端口</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEdit_srvIp</cstring>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_srvConnect">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>88</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_srvDisConnect">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>88</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>断开</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>210</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>角度</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Base角度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEdit_srvIp</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>57</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Neck角度</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEdit_srvIp</cstring>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>13</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.375000000000000</double>
     </property>
     <property name="value">
      <double>90.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.900000000000000</double>
     </property>
     <property name="value">
      <double>90.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_baseAngelRun">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>11</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>运行到</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_neckAngleRun">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>运行到</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>309</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>图片剪裁</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>62</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CropX">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>661</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>图片预览</string>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>640</width>
       <height>360</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>8</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>状态</string>
    </property>
    <widget class="QLabel" name="label_connState">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>未连接</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_serial">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>28</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设备序号：000000000000</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_srvIp</tabstop>
  <tabstop>lineEdit_srvPort</tabstop>
  <tabstop>pushButton_srvConnect</tabstop>
  <tabstop>pushButton_srvDisConnect</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>doubleSpinBox_2</tabstop>
  <tabstop>pushButton_baseAngelRun</tabstop>
  <tabstop>pushButton_neckAngleRun</tabstop>
  <tabstop>lineEdit_CropX</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
